Bio/Blast/NCBIXML.py

```diff
@@ -252,53 +252,53 @@ def _end_BlastOutput_version(self):
                 self._header.date = parts[2]
 
     def _end_BlastOutput_reference(self):
-        """A reference to the article describing the algorithm (PRIVATE).
+        """Yield the article describing the algorithm (PRIVATE).
 
         Save this to put on each blast record object
         """
         self._header.reference = self._value
 
     def _end_BlastOutput_db(self):
-        """The database(s) searched (PRIVATE).
+        """Yield the database(s) searched (PRIVATE).
 
         Save this to put on each blast record object
         """
         self._header.database = self._value
 
     def _end_BlastOutput_query_ID(self):
-        """The identifier of the query (PRIVATE).
+        """Yield the identifier of the query (PRIVATE).
```

Comment:
Similar comment to above, how about  ``Record the identifier of the query (PRIVATE).`` instead?